earth_enterprise/rpms/build.gradle

```diff
@@ -272,15 +318,26 @@ task openGeeCommonDeb (type: Deb) {
     }
 }
 
-task openGeeServerRpm (type: Rpm, dependsOn: openGeeSharedFiles) {
+task openGeeServerRpm (type: TstRpm, dependsOn: openGeeSharedFiles) {
```

Comment:
Why do these begin with Tst? Shouldn't it be something like GeeRpm?

Reply:
This a class derived from `com.netflix.gradle.plugins.rpm.Rpm` from the OS Package plug-in.  It extends the `Rpm` Gradle task to be able to add commands as dependencies, and scan files that go into the RPM for shared library, and scripting language module dependencies, as well as list shared libraries that go into the package as `provides`.

Reply:
We could submit those in a pull request to OS Package, except the latest version of OS Package fails to produce RPMs that install on Cent OS 6, so we'd have to get it into an earlier version, and propagated to the latest version for whet it, hopefully, gets fixed to work with Cent OS / RHEL 6.

Reply:
@tst-ccamp: Oh, you're right.  We may not want to have that in OpenGEE.  Do we have reverse domain name prefix we can use for OpenGEE packages?